### PR TITLE
US273775: Add platform events

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

US273775: Add platform events

### Description of the Change

Added platform event handler
Modified service configuration module
Modified platform events

### CODE REVIEW

File: platform_events-handler/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a sample NodeJs template project
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events-handler/README.md]: In cooperated review comments

File: platform_events-handler/deployment-env.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+service : get-events-handler
+domain : jazz
+owner: Smalire
+securityGroupIds: sg-5a1e9023
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events-handler/deployment-env.yml]: In cooperated review comments. All these values will replaced with service_id while building the stack

File: platform_events-handler/deployment-env.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+subnetIds: subnet-5b15363e, subnet-6f135218, subnet-71a63e28
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events-handler/deployment-env.yml]: In cooperated review comments. All these values will replaced with service_id while building the stack

File: platform_events-handler/deployment-env.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+providerRuntime : nodejs4.3
+providerMemorySize : 256
+providerTimeout : 160
+region: us-west-2
+# Put your custom role here which has the IAM permissions required by your code
+iamRoleARN: arn:aws:iam::302890901340:role/jazz_platform_services
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events-handler/deployment-env.yml]: In cooperated review comments. All these values will replaced with service_id while building the stack

File: platform_events/components/config.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // =========================================================================
-// Copyright � 2017 T-Mobile USA, Inc.
-// 
+// Copyright ? 2017 T-Mobile USA, Inc.
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events/components/config.js]: In cooperated review comments.

File: platform_events/components/config.js [hunk 6]
```diff
@@ -28,11 +28,13 @@ var getStageConfig = (event) => {
...
+        configObj = require('../config/dev-config.json');
     } else if (stage === 'stg') {
-        configObj = require('./stg-config.json');
-    } else {
-        configObj = require('./prod-config.json');
+        configObj = require('../config/stg-config.json');
+    } else if (stage === 'prod'){
+        configObj = require('../config/prod-config.json');
+    }else {
+        configObj = require('../config/' + stage + '-config.json')
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events/components/config.js]: In cooperated review comments.

File: platform_events-handler/test/test.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const assert = require('chai').assert;
```
[the remark this answers is not in the document]
Comment (SiniWilson) [reply platform_events-handler/test/test.js]: Test case is handling in separate story my belief.

File: platform_events-handler/.gitignore [hunk 8]
```diff
@@ -0,0 +1,4 @@
+# Add any directories, files, or patterns you don't want to be tracked by version control
```
Comment (suryajak) [inline platform_events-handler/.gitignore]: This file needs to be removed. We have a gitignore at root.

File: platform_events-handler/components/config.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
+var getStageConfig = (context) => {
```
Comment (suryajak) [inline platform_events-handler/components/config.js]: We have a newer version of this file (you have referenced it down below). To be consistent can you replace this with that?

File: platform_events-handler/index.js [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+	var errorHandler = errorHandlerModule(logger);
+	const dynamodb = new AWS.DynamoDB();
+	var processedEvents = [];
+	var failedEvents = [];
+
+	var docClient = new AWS.DynamoDB.DocumentClient();
+	var interestedEvents = [];
+	var failureCodes = fcodes();
+	
+	async.series({
```
Comment (suryajak) [inline platform_events-handler/index.js]: We want to use promises native for all new code to OSS. Can you attempt to convert this to the OSS model code? If you are having to spend more than 1 hour for this, we will revise your estimates.

**Pull request merged**